wasm/src/lib.rs

```diff
@@ -1,28 +1,36 @@
+mod wasm_builtins;
+
 extern crate rustpython_vm;
 extern crate wasm_bindgen;
-use rustpython_vm::compile;
+extern crate web_sys;
+
 use rustpython_vm::VirtualMachine;
+use rustpython_vm::compile;
+use rustpython_vm::pyobject::AttributeProtocol;
+use rustpython_vm::import::import_module;
+use std::path::PathBuf;
 use wasm_bindgen::prelude::*;
-
-#[wasm_bindgen]
-extern "C" {
-    // Use `js_namespace` here to bind `console.log(..)` instead of just
-    // `log(..)`
-    #[wasm_bindgen(js_namespace = console)]
-    fn log(s: &str);
-}
+use web_sys::console;
 
 #[wasm_bindgen]
 pub fn run_code(source: &str) -> () {
     //add hash in here
-    log("Running RustPython");
-    log(&source.to_string());
+    console::log_1(&"Running RustPython".into());
+    console::log_1(&"Running code:".into());
+    console::log_1(&source.to_string().into());
+    console::log_1(&"----- console -----".into());
+
     let mut vm = VirtualMachine::new();
+
     let code_obj = compile::compile(&mut vm, &source.to_string(), compile::Mode::Exec, None);
     let builtins = vm.get_builtin_scope();
+    //let ctx = vm.context();
+    //let current_path = PathBuf::from(".");
+    //let builtins = import_module(&mut vm, current_path, "builtins").unwrap();
+    builtins.set_attr("print", vm.context().new_rustfunc(wasm_builtins::builtin_print));
```

Comment:
@windelbouwman could you kindly help me check this part? It compiles correctly but seems to panic in runtime. 

The browser gives this error: 
```
bootstrap.js:5 Error importing `index.js`: RuntimeError: unreachable
    at __rust_start_panic (wasm-function[3157]:1)
    at rust_panic (wasm-function[3135]:31)
    at std::panicking::rust_panic_with_hook::ha9514ccd859407a0 (wasm-function[3130]:305)
    at std::panicking::continue_panic_fmt::hbdf64f9b91d66da7 (wasm-function[3129]:120)
    at std::panicking::begin_panic_fmt::hb1f86670af92c2cf (wasm-function[3067]:95)
    at _$LT$alloc..rc..Rc$LT$core..cell..RefCell$LT$rustpython_vm..pyobject..PyObject$GT$$GT$$u20$as$u20$rustpython_vm..pyobject..AttributeProtocol$GT$::set_attr::h29a4da0736316e29 (wasm-function[587]:367)
    at rustpython_wasm::run_code::h6cdd6ee5f53d3532 (wasm-function[5]:344)
    at run_code (wasm-function[6]:16)
    at Module.run_code (webpack:///../pkg/rustpython_wasm.js?:34:76)
    at eval (webpack:///./index.js?:5:57)
```